core/state-machine/src/ext.rs

```diff
@@ -181,35 +181,41 @@ where
 	H::Out: Ord + HeapSizeOf,
 {
 	fn storage(&self, key: &[u8]) -> Option<Vec<u8>> {
+		let _guard = panic_handler::AbortGuard::new(true);
```

Comment:
Why do we need to abort here?

Reply:
As @rphmeier answered in the wrong comment, client code panic should not lead to runtime error and potential consensus issues. Also panic behaviour between native and wasm runtimes needs to be consistent.

Reply:
Okay. :)